examples/cassandra/README.md

```diff
@@ -57,103 +71,51 @@ command line tool somewhere in your path.  Please see the
 for installation instructions for your platform.
 
 This example also has a few code and configuration files needed.  To avoid
-typing these out, you can `git clone` the Kubernetes repository to you local
+typing these out, you can `git clone` the Kubernetes repository to your local
 computer.
 
-### A note for the impatient
+## tl;dr Quickstart
 
-This is a somewhat long tutorial.  If you want to jump straight to the "do it
-now" commands, please see the [tl; dr](#tl-dr) at the end.
+If you want to jump first straight to the commands we will run,
+here are the steps:
 
-### Simple Single Pod Cassandra Node
+```sh
+# create a service to track all cassandra nodes
+kubectl create -f examples/cassandra/cassandra-service.yaml
```

Comment:
That sounds good to me too, but I'd like to do that in a following PR, as I'm having enough trouble chasing down the cascade of test errors caused by simply removing cassandra.yaml.  (I didn't write the tests myself).